Lib/idlelib/autocomplete.py

```diff
@@ -90,6 +92,126 @@ def _delayed_open_completions(self, args):
+        if hp.is_in_string():
+            opening_bracket_idx = hp.bracketing[hp.indexbracket - 1][0]
+            if hp.rawtext[opening_bracket_idx] != '[':
+                return None
+            opening_quote_idx = hp.bracketing[hp.indexbracket][0]
+            m = cls._dict_str_key_prefix_re.fullmatch(
+                hp.rawtext[opening_bracket_idx+1:opening_quote_idx])
+            if m is None:
+                return None
+            comp_start = m.group(1) + hp.rawtext[opening_quote_idx:hp.indexinrawtext]
+        elif hp.is_in_code():
+            if not hp.isopener[hp.indexbracket]:
+                return None
+            opening_bracket_idx = hp.bracketing[hp.indexbracket][0]
+            if hp.rawtext[opening_bracket_idx] != '[':
+                return None
+            m = cls._dict_str_key_prefix_re.fullmatch(
+                hp.rawtext[opening_bracket_idx + 1:hp.indexinrawtext])
+            if m is None:
+                return None
+            comp_start = m.group(1)
+        else:
+            return None
+
+        hp.set_index("insert-%dc" % (len(hp.rawtext) - opening_bracket_idx))
+        comp_what = hp.get_expression()
+        return comp_what, comp_start
+
+    @classmethod
+    def dict_key_reprs(cls, comp_start, comp_list):
+        """Customized repr() for str and bytes.
+        
+        This retains the prefix and quote type of the completed partial
+        str/bytes literal, if they exist.
+        """
+        # Analyze existing prefix and quote type of the completion start.
+        is_raw = False
+        quote_type = '"'
+        prefix_match = cls._dict_str_key_prefix_re.match(comp_start)
+        if prefix_match is not None:
+            prefix = prefix_match.group(1)
+            post_prefix = comp_start[prefix_match.end():]
+            lowercase_prefix = prefix.lower()
+
+            is_raw = 'r' in lowercase_prefix
+            if post_prefix:
+                quote_type_candidate = post_prefix[0]
+                if quote_type_candidate not in "'\"":
+                    pass  # use the default quote type
+                elif post_prefix[1:2] == quote_type_candidate:
+                    quote_type = quote_type_candidate * 3
+                else:
+                    quote_type = quote_type_candidate
+        else:
+            prefix = lowercase_prefix = ''
+
+        # Prepare the prefixes to use for the reprs.
+        prefix = prefix.replace('b', '').replace('B', '')
+        bytes_prefix = (prefix + 'b') if lowercase_prefix in ('', 'r') else 'b'
+
+        # Prepare escaping utilities.
+        escape_re = re.compile(r'[\0-\15]')
+        escapes = {chr(x): f'\\{x}' for x in range(7)}
+        escapes.update({
+            7: '\\a',
+            8: '\\b',
+            9: '\\t',
+            10: '\\n',
+            11: '\\v',
+            12: '\\f',
+            13: '\\r',
+        })
+        high_byte_escape_re = re.compile(r'[\x80-\xff]')
```

Comment:
Based on the [travis error log](https://travis-ci.org/python/cpython/jobs/569053155), it looks like this line is like a culprit (missing empty line before function):
```suggestion
        high_byte_escape_re = re.compile(r'[\x80-\xff]')
        
```